File: p17_u3.cs
using System;

namespace U3programa17 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            char salir;
            
            do
            {
                Console.WriteLine("Bienvenido a una sala de ejecucion");
                Console.WriteLine("Escriba n para no condenar a muerte");
                salir = Convert.ToInt32(Console.ReadLine()).charAt(0);
            } while (salir != 'n');
        }
    }
}

File: P17_u3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p17_u3;
import java.util.Scanner;
/**
 *
 * @author richy
 */
public class P17_u3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        char salir;

        do
        {
            System.out.println("Bienvenido a una sala de ejecucion");
            System.out.println("Escriba n para no condenar a muerte");
            salir = sc.next().charAt(0);
    }while(salir !='n');
    }
}
